src/lib.rs

```diff
@@ -198,6 +199,20 @@ pub struct Config {
     #[builder(default)]
     /// The version of Synapse to use
     pub synapse: SynapseVersion,
+
+    #[serde(default)]
+    #[builder(default)]
+    /// Information for logging to a registry.
+    ///
+    /// May be overridden from the command-line.
```

Comment:
Does the readme say how to do override from the command-line? I assume that you almost always want to do that when working with CI?

Reply:
Fixed that!